invenio_communities/assets/semantic-ui/js/invenio_communities/subcommunity/new.js

```diff
@@ -150,163 +150,162 @@ class CommunityCreateForm extends Component {
-                    {i18next.t("Subcommunity request")}
-                  </Header>
-                  <Divider />
-                </Grid.Column>
-              </Grid.Row>
-              <Grid.Row textAlign="left">
-                <Grid.Column mobile={16} tablet={12} computer={8}>
-                  <div className="field">
-                    <Form.Field>
-                      {i18next.t("Do you already have an existing community?")}
-                    </Form.Field>
-                    {/* <Form.Group aria-labelledby="community-label"> */}
-                    <Form.Group>
-                      <RadioField
-                        label={i18next.t("Yes")}
-                        checked={hasCommunity === true}
-                        value={i18next.t("Yes")}
-                        onChange={() => {
-                          this.setState({ hasCommunity: true });
-                        }}
-                        fieldPath="metadata.hasCommunity"
-                      />
-                      <RadioField
-                        label={i18next.t("No")}
-                        checked={hasCommunity === false}
-                        value={i18next.t("No")}
-                        onChange={() => {
-                          this.setState({ hasCommunity: false });
-                        }}
-                        fieldPath="metadata.hasCommunity"
-                      />
-                    </Form.Group>
-                  </div>
-                  {hasCommunity && (
-                    <SelectField
-                      label={
-                        <FieldLabel
-                          icon="user"
-                          label={i18next.t("Community")}
-                          id="community-label"
-                          class="block"
+                <Grid.Row>
+                  <Grid.Column mobile={16} tablet={12} computer={8} textAlign="center">
+                    <Header as="h1" className="rel-mt-2">
+                      {i18next.t("Subcommunity request")}
+                    </Header>
+                    <Divider />
+                  </Grid.Column>
+                </Grid.Row>
+                <Grid.Row textAlign="left">
+                  <Grid.Column mobile={16} tablet={12} computer={8}>
+                    <div className="field">
+                      <Form.Field>
+                        {i18next.t("Do you already have an existing community?")}
+                      </Form.Field>
+                      {/* <Form.Group aria-labelledby="community-label"> */}
+                      <Form.Group>
+                        <RadioField
+                          label={i18next.t("Yes")}
+                          checked={hasCommunity === true}
+                          value={i18next.t("Yes")}
+                          onChange={() => {
+                            this.setState({ hasCommunity: true });
+                          }}
+                          fieldPath="metadata.hasCommunity"
```

Comment:
discussed IRL: we can use `Radio` (From semantic UI react) since this component only controls the state `hasCommunity` and is not linked at all with the form itself. 

To be tackled in UI improvements.